Resources/config/services.yml

```diff
@@ -1,38 +1,21 @@
-parameters:
-    liuggio_stats_d_client.service.class: Liuggio\StatsdClient\StatsdClient
-    liuggio_stats_d_client.entity.class: Liuggio\StatsdClient\Entity\StatsdData
-    liuggio_stats_d_client.factory.class: Liuggio\StatsdClient\Factory\StatsdDataFactory
-    liuggio_stats_d_client.sender.debug.class: Liuggio\StatsdClient\Sender\EchoSender
-    liuggio_stats_d_client.collector.service.class: Liuggio\StatsDClientBundle\Service\StatsDCollectorService
-    liuggio_stats_d_client.collector.listener.class: Liuggio\StatsDClientBundle\Listener\StatsDCollectorListener
-    liuggio_stats_d_client.collector.dbal.class: Liuggio\StatsDClientBundle\StatsCollector\DbalStatsCollector
-    liuggio_stats_d.sender.class: Liuggio\StatsdClient\Service\StatsdService
-
-# monolog handler
-    liuggio_stats_d_client.monolog_handler.class: Liuggio\StatsdClient\Monolog\Handler\StatsDHandler
-
 services:
 # statsd-php-client
     liuggio_stats_d_client.factory:
-        class: '%liuggio_stats_d_client.factory.class%'
-        arguments: ['%liuggio_stats_d_client.entity.class%']
-
-    liuggio_stats_d_client.sender.service:
```

Comment:
The class parameter was dynamic because of the `debug` flag, by simplifying  and having only one class parameter it didn't make sense to write the configuration (thus, cf previous comment)